pages/awards/[id].js

```diff
@@ -0,0 +1,236 @@
+
+// Page: Award Details
+export default function AwardDetails() {
+  const [selectedTab, setSelectedTab] = useState(-1);
+  const [showAwardSummary, setShowAwardSummary] = useState(true);
+  const [feedbackAvailable, setFeedbackAvailable] = useState(true);
+
+  const handleTabClick = () => {
+    setFeedbackAvailable(!feedbackAvailable); // TODO: Change this!!
+    setShowAwardSummary(false);
+  };
+
+  return (
+    <Layout>
+      <div>
+        <div>
+          <h2 className={styles.performances_details__eventName}>{`OSSDF2021- Let's Dis-dance`}</h2>
+        </div>
+        <div>
+          <Title
+            className={styles.performances__header__pageTitle}
+          >{`Most Inspiring Medium Group Performance Awards`}</Title>
+        </div>
+        <div className={styles.performance_details__content_container}>
+          <div className={styles.performance_details__tabs_container}>
+            <div
+              className={`${styles.performance_details__tabs} ${
+                showAwardSummary && styles.performance_details__tabs_selected
+              }`}
+            >
+              <button
+                onClick={() => {
+                  setShowAwardSummary(true);
+                  setFeedbackAvailable(true);
+                  setSelectedTab(-1);
+                }}
+              >
+                <h3>Award Details</h3>
+              </button>
+            </div>
+            <Tab
+              selected={selectedTab === 0}
+              setSelectedTab={setSelectedTab}
+              tabIndex={0}
+              handleClick={handleTabClick}
+              nominations={`1/3 Judges`}
+            >
+              You’re The One That I Want
+            </Tab>
+            <Tab
+              selected={selectedTab === 1}
+              setSelectedTab={setSelectedTab}
+              tabIndex={1}
+              handleClick={handleTabClick}
+              nominations={`3/3 Judges`}
+            >
+              You’re The One That I Want
+            </Tab>
+            <Tab
+              selected={selectedTab === 2}
+              setSelectedTab={setSelectedTab}
+              tabIndex={2}
+              handleClick={handleTabClick}
+              nominations={`2/3 Judges`}
+            >
+              You’re The One That I Want
+            </Tab>
+          </div>
+          <div
+            className={`${styles.performance_details__content} ${
+              !feedbackAvailable && styles.performance_details__no_content_container
+            }`}
+          >
+            {showAwardSummary ? (
+              <AwardSummary />
+            ) : feedbackAvailable ? (
+              <JudgeFeedback />
+            ) : (
+              <EmptyComponent />
+            )}
+          </div>
+        </div>
+      </div>
+    </Layout>
+  );
+}
+
+function Tab(props) {
+  const handleClick = () => {
+    props.setSelectedTab(props.tabIndex);
+    props.handleClick();
+  };
```

Comment:
Can we destructure props here so we can use PropTypes? We can also do it during the API hookup.